diff --git a/target/i386/hvf/hvf.c b/target/i386/hvf/hvf.c
index d72543dc31..ece6f3a6fd 100644
--- a/target/i386/hvf/hvf.c
+++ b/target/i386/hvf/hvf.c
@@ -600,7 +600,10 @@ int hvf_init_vcpu(CPUState *cpu)
           VMCS_PRI_PROC_BASED_CTLS_SEC_CONTROL);
     wvmcs(cpu->hvf_fd, VMCS_SEC_PROC_BASED_CTLS,
           cap2ctrl(hvf_state->hvf_caps->vmx_cap_procbased2,
-                   VMCS_PRI_PROC_BASED2_CTLS_APIC_ACCESSES));
+                   VMCS_PRI_PROC_BASED2_CTLS_APIC_ACCESSES |
+                   CPU_BASED2_RDTSCP |
+                   CPU_BASED2_INVPCID |
+                   CPU_BASED2_XSAVES_XRSTORS));
 
     wvmcs(cpu->hvf_fd, VMCS_ENTRY_CTLS, cap2ctrl(hvf_state->hvf_caps->vmx_cap_entry,
           0));
